README.md

```diff
@@ -79,8 +79,8 @@ A new package can be created using `psc-package init`. This will:
 
 To add a dependency, either:
 
-- Use the `install` command, which will update the project configuration automatically, or
-- Modify the `psc-package.json` file, and sync manually by running the `update` command.
+- Use the `install <package name>` command, which will update the project configuration automatically, or
+- Modify the `psc-package.json` file, and sync manually by running the `install` command.
```

Comment:
Not sure if it's worth commenting "(previously `update`)" or that just confuses matters